src/frontend/src/components/Post/Post.jsx

```diff
@@ -141,7 +152,12 @@ const Post = ({ postUrl }) => {
       <ListSubheader className={classes.header}>
         <AdminButtons />
         <Typography variant="h1" title={post.title} id={post.id} className={classes.title}>
-          {post.title}
+          <span
+            onClick={toggleExpandHeader}
```

Comment:
Let's inline this function, since it's so short:

```js
onClick={() => setExpandHeader(!expandHeader)}
```

And you can drop `toggleExpandHeader` above.